### PR TITLE
Notes endpoints

### PR DESCRIPTION
**Pull request by Jdww**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -8113,11 +8113,13 @@ Fetch cloudPlatform url for type and id
...
 ## NoteSubjectTypes (enum)
-+ `contact`
 + `company`
++ `contact`
++ `creditNote`
 + `deal`
 + `invoice`
-+ `creditNote`
-+ `subscription`
++ `order`
```
Comment (tiagobastos) [inline apiary.apib]: `order` is not an API v2 concept yet. 
No need to include it if we have no way to find UUIDs with that note subject type.

**Review by joaoinacio — DISMISSED** [review]

File: src/01-general/notes.apib [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+    + Attributes (object)
+        + data (array)
+            + (object)
+                + id: `a344c251-2494-0013-b433-ccee8e8435e5` (string)
+                + content: `new note content` (string)
+                + subject (object)
+                    + type: `contact` (NoteSubjectTypes)
+                    + id: `36386b05-936e-4cc0-9523-bd20d797ebf5` (string)
+                + added_at: `2016-01-01T00:00:00+00:00` (string)
```
Comment (gdegirmenci) [inline src/01-general/notes.apib]: Eol

**Review by joaoinacio — DISMISSED** [review]

**Review by gdegirmenci — DISMISSED** [review]

File: apiary.apib [hunk 3]
```diff
@@ -1539,6 +1540,68 @@ Get a list of exchange rates for a provided currency.
...
+                + id: `a344c251-2494-0013-b433-ccee8e8435e5` (string)
+                + content: `new note content` (string)
+                + subject (object)
+                    + type: `contact` (NoteSubjectTypes)
+                    + id: `36386b05-936e-4cc0-9523-bd20d797ebf5` (string)
+                + added_at: `2016-01-01T00:00:00+00:00` (string)
+
+### notes.create [POST /notes.create]
+
+Add a new create.
```
Comment (joaoinacio) [inline apiary.apib:1572]: 👀

Comment (joaoinacio) [inline apiary.apib]:
```suggestion
Create a new note.
```

**Review by joaoinacio — DISMISSED** [review]

**Comment by joaoinacio** [comment]
✅

File: apiary.apib [hunk 4]
```diff
@@ -1539,6 +1540,68 @@ Get a list of exchange rates for a provided currency.
...
++ Response 201 (application/json)
+
+    + Attributes (object)
+        + data (object)
+            + type: `note` (string)
+            + id: `ee94b4c0-5786-0517-9d26-8dd1e6406f20` (string)
+
+### notes.update [POST /notes.update]
+
+Update an existing note
```
Comment (gdegirmenci) [inline apiary.apib]:
```suggestion
Update an existing note.
```

**Review by tiagobastos — DISMISSED** [review]

**Review by joaoinacio — APPROVED** [review]

**Review by gdegirmenci — APPROVED** [review]

**Review by tiagobastos — APPROVED** [review]

**Pull request merged**